backend/tests/adapters/message_bus.py

```diff
@@ -75,5 +84,5 @@ async def rpc(self, message: InfrahubMessage, response_class: type[ResponseClass
     def seen_routing_keys(self) -> list[str]:
         return list(self.messages_per_routing_key.keys())
 
-    async def initialize(self, service: InfrahubServices) -> None:
-        self.service = service
+    async def shutdown(self) -> None:
+        pass
```

Comment:
I think having to add these comes from the change to abstract base classes. I don't quite see the value of that. I think it's cleaner to keep the the classes as just regular base classes and then we can have a silent shutdown if the child adapter doesn't need to do anything there's nothing to implement.

Reply:
Ok, initial reason was that shutdown is always defined for production bus (`RabbitMQ`, `Nats`), so I feel it makes sense to explicitly override `shutdown` for testing bus. But your point makes sense too I'll let it as is